src/parse/get_docstring_type.ts

```diff
@@ -0,0 +1,18 @@
+export function getDocstringType(functionDefinition: string, linePosition: number): string {
```

Comment:
I think we should turn the return type into a string literal type so that we restrict to only types we define.